### DISCUSSION TITLE
Is Flower being actively maintained in 2024?

### DISCUSSION THREAD

**Discussion by dpdoughe** [mher/flower#1368] [General]
Sorry for this awkward question. I've noticed the last pull request to Flower to be merged was in Dec, 2023 and I noticed @mher 's account doesn't show any GitHub activity in 2024. Is there a different maintainer we should be pinging for my pull request reviews? What is the expected cadence for pull request reviews?

**Comment by martinlehoux** [comment]
Maybe it's time to create a fork? I can't see any fork that seems widely starred in https://github.com/mher/flower/forks

It could easily be merged back to this one if activity is up

**Reply by kgk** [reply]
I would good if you could still push to pypi:  https://pypi.org/project/flower/
We usually install it from there.

**Comment by adamcadaver** [comment]
It's a year later and I just made an issue with the same question. This repo is not being worked on. I think we need to work with the celery team to point to a fork of this repo that is going to be actively maintained.

**Comment by kgk** [comment]
Maybe write the celery maintainers.. it probably belongs under their umbrella

**Reply by adamcadaver** [reply]
https://github.com/celery/celery/issues/9814

**Reply by auvipy** [reply]
we can maintain a fork

**Comment by Nusnus** [comment]
FYI https://github.com/celery/celery/issues/9814#issuecomment-3153837554

**Comment by Nusnus** [comment]
@dpdoughe 
> Sorry for this awkward question. I've noticed the last pull request to Flower to be merged was in Dec, 2023 and I noticed @mher 's account doesn't show any GitHub activity in 2024. Is there a different maintainer we should be pinging for my pull request reviews? What is the expected cadence for pull request reviews?

[Yes](https://github.com/celery/celery/issues/9814#issuecomment-3172617334).

**Reply by auvipy** [reply]
we the celery core team joined in maintenance team of flower.
